assets/js/components/Table/EmptyTable.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames';
+
+function EmptyTable({ colSpan = 1, className = '', fontSize = 'text-sm' }) {
```

Comment:
Since we are not using this prop externally and there's no way to use this explicitly, you could just remove the `fontSize` and `className` props

Reply:
I deleted the EmptyTable story as the Table component already has it covered!